site/src/Librecores/ProjectRepoBundle/Validator/Constraints/UserOrgNameValidator.php

```diff
@@ -76,7 +76,7 @@ class UserOrgNameValidator extends ConstraintValidator
      *
      * @var bool
      */
-    const CHECK_UNIQUE_ORG  = true;
+    const CHECK_UNIQUE_ORG  = false;
```

Comment:
The unique username check results to the generation of two constraint validation warnings on the user's form and that's why it was originally removed from here. Thus we must keep either the unique check on the entity (doctrine annotation) or here but not both. Maybe there is another solution that keeps both but only shows one on the form?, I haven't searched for that though, maybe it's possible, in relation to constraint validation groups, see below.

The unique org name check was removed because keeping it makes it trigger everytime you edit the organization settings and just keep / submit the same name (99.9% of cases). I've tried removing the field from the form as to disallow the editing of the name by users altogether, like we do on projects, but it keeps triggering. We can solve this only by having constraint validation groups that trigger the constraints on selected CRUD operations. Thus we should trigger certain constraints only on org creation but not on org updates etc. Maybe leave it as is now and make this in a future change-set that is aimed at having more complete constraints?